<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddItem</class>
 <widget class="QDialog" name="AddItem">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add an Item</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhExclusiveInputMask</set>
  </property>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtName">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtQuantity">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDOP">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date of Purchase</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDOS">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date of Sell</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSellingPrice">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>238</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Selling Price</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPurchasePrice">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Purchase Price</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbStatus">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Available</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Expired</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sold</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Return</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Spoiled</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lblInfo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <italic>true</italic>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Enter The Details of your item:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
